spec/features/gardens/index_spec.rb

```diff
@@ -1,20 +1,56 @@
 require 'rails_helper'
 
 RSpec.describe 'Index Page' do
```

Comment:
Do we need this file since we removed gardens index?

Reply:
Yeah let's get rid of it.  Dream driving can get too dreamy ;)